express/scripts/scripts.js

```diff
@@ -109,6 +109,57 @@ export function getMeta(name) {
   return value;
 }
 
+// Get lottie animation HTML - remember to lazyLoadLottiePlayer() to see it.
+export function getLottie(name, src, loop = true, autoplay = true, control = false, hover = false) {
+  return (`<lottie-player class="lottie lottie-${name}" src="${src}" background="transparent" speed="1" ${(loop) ? 'loop ' : ''}${(autoplay) ? 'autoplay ' : ''}${(control) ? 'controls ' : ''}${(hover) ? 'hover ' : ''}></lottie-player>`);
+}
+
+// Lazy-load lottie player if you scroll to the block.
+export function lazyLoadLottiePlayer($block = null) {
+  const loadLottiePlayer = () => {
+    if (window['lottie-player']) return;
+
+    const script = document.createElement('script');
+    script.type = 'text/javascript';
+    script.async = true;
+    script.src = 'https://unpkg.com/@lottiefiles/lottie-player@latest/dist/lottie-player.js';
```

Comment:
```suggestion
    script.src = '/express/scripts/lottie-player.1.5.6.js';
```
We should now use `/express/scripts/lottie-player.1.5.6.js` instead of loading the external file.